apps/condo/domains/news/schema/ExportNewsRecipientsService.js

```diff
@@ -0,0 +1,242 @@
+/**
+ * Generated by `createservice news.ExportNewsRecipientsService --type mutations`
+ */
+
+const dayjs = require('dayjs')
+const get = require('lodash/get')
+const isEmpty = require('lodash/isEmpty')
+const isNil = require('lodash/isNil')
+
+const conf = require('@open-condo/config')
+const { GQLError, GQLErrorCode: { INTERNAL_ERROR } } = require('@open-condo/keystone/errors')
+const { GQLCustomSchema } = require('@open-condo/keystone/schema')
+const { extractReqLocale } = require('@open-condo/locales/extractReqLocale')
+const { i18n } = require('@open-condo/locales/loader')
+
+const { NOT_FOUND } = require('@condo/domains/common/constants/errors')
+const { createExportFile } = require('@condo/domains/common/utils/createExportFile')
+const { getHeadersTranslations, EXPORT_TYPE_NEWS_RECIPIENTS } = require('@condo/domains/common/utils/exportToExcel')
+const { loadListByChunks } = require('@condo/domains/common/utils/serverSchema')
+const access = require('@condo/domains/news/access/ExportNewsRecipientsService')
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
+const { Property } = require('@condo/domains/property/utils/serverSchema')
+const { Resident } = require('@condo/domains/resident/utils/serverSchema')
+
+
+const getUnitsFromProperty = ({ map }) => (
+    map?.sections?.reduce((acc, section) => ([
```

Comment:
We're trying not to use optional chaining syntax at all. You can rewrite this expression with utils from lodash